README.md

```diff
@@ -83,6 +83,62 @@ class SomeModel extends Model {
 }
 ``` 
 
+### CamelCaseModel
+
+Extended class Model for use camel case notation in DB.
+
+**Example**:
+
+```php
+use Saritasa\Database\Eloquent\Models\CamelCaseModel;
+
+class SomeModel extends CamelCaseModel
+{
+    //your code
+}
+```
+
+### CamelCaseBlueprint
+
+Extends class Blueprint for use camel case in base migration methods
+
+**Example**:
+```php
+use Illuminate\Database\Migrations\Migration;
+use Saritasa\Database\Schema\CamelCaseBlueprint as Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSomeTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('someTable', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('someColumnName');
+            $table->timestamps();
+        });
+    }
+}
+```
+
+Note: ```$table->timestamps();``` will create columns with names "createdAt" and
+"UpdatedAt".
+
+### CamelCaseForeignKeys trait
+
+Use in any model class for get the default foreign key name for this model.
+
+**Example**:
+
+```php
+use Saritasa\Database\Eloquent\Models\CamelCaseForeignKeys;
+
+class MyModel extends SomeModelClass
+{
+    use CamelCaseForeignKeys;
+}
+```
+
```

Comment:
Add some relation and foreign key example - like above wtith createdAt and cpdatedAt - difference between original and modified behavior.